hs/app/reach/embed/sh/subcommand/scaffold/docker-compose.yml

```diff
@@ -0,0 +1,86 @@
+version: '3.4'
+x-app-base: &app-base
+  image: ${IMAGE_TAG}
+services:
+  devnet-cfx:
```

Comment:
I imagine that each connector will have a single `docker-compose.yml` excerpt that we maintain in one place and plop in when needed. Meaning that this file would be generated by splicing all of those together